### DISCUSSION TITLE
Electron Reloading

### DISCUSSION THREAD

**Discussion by ghost** [nklayman/vue-cli-plugin-electron-builder#1670] [Q&A] [answered]
Hi,

I currently use the Vue CLI Plugin Electron Builder package and I'm wondering if it's possible to have electron reload instead of completely restarting it once you serve files with Vue.

I've basically added my entire src directory here: https://github.com/nklayman/vue-cli-plugin-electron-builder/issues/741

So, now when I change something in background.js or another related file, it will reload the vue development server, but I have to completely restart Electron to be able to see the changes. It would be nice to just keep Electron open instead of restarting it every time.

Thanks!

**Comment by klren0312** [comment, marked as answer]
When you change the mainProcess file (like background.js or another related file), Electron must be restart

**Reply by ghost** [reply]
> When you change the mainProcess file (like background.js or another related file), Electron must be restart

Gotcha, thanks for verifying!
